src/tribler-core/tribler_core/modules/metadata_store/orm_bindings/metadata_node.py

```diff
@@ -109,6 +110,7 @@ def get_entries_query(
             pony_query = pony_query.where(lambda g: g.status != TODELETE) if exclude_deleted else pony_query
             pony_query = pony_query.where(lambda g: g.xxx == 0) if hide_xxx else pony_query
             pony_query = pony_query.where(lambda g: g.status != LEGACY_ENTRY) if exclude_legacy else pony_query
+            pony_query = pony_query.where(lambda g: g.infohash == infohash) if infohash else pony_query
```

Comment:
It is more correct to use `if infohash is not None` because otherwise, you will not be able to query on _empty string_ infohashes.

Reply:
Agree, but what is wrong with querying an empty string infohashes?

Reply:
There is nothing wrong on querying an empty string infohashes. The problem is, in Python both empty string and `None` is considered `False` in conditionals. So you should be very precise about it.

Reply:
Okay, that's a good point, thank you!